common/src/main/java/io/bisq/common/locale/CurrencyUtil.java

```diff
@@ -88,8 +88,8 @@ public static void setBaseCurrencyCode(String baseCurrencyCode) {
         final List<CryptoCurrency> result = new ArrayList<>();
 
         if (DevEnv.DAO_TRADING_ACTIVATED)
-            result.add(new CryptoCurrency("BSQ", "Bisq Token"));
-
+            result.add(new CryptoCurrency("AXE", "Axe"));
```

Comment:
This doesn't look right... please revert that change.
Instead, AXE should be added a few lines down, just above this:
> result.add(new CryptoCurrency("BCH", "Bitcoin Cash"));

Reply:
got it, thought its maybe alphabet order